app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -107,9 +107,29 @@ def textual_endpoints
       :value => @record.connection_configurations.hawkular.endpoint.port}]
   end
 
+  def redact_passwords_from_proxy_values(name, value)
+    unless ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::PROXY_ENV_VARIABLES.include?(name)
```

Comment:
actually, if other custom attrs contain URL with user/pwd, perhaps these are also worth redacting?
(no opinion, just throwing the idea out)